EdurecaInClassProject/src/app/components/error/error.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="container box">
+    <p class="alert alert-danger"> Path not found</p>
```

Comment:
Do not Hardcode values, store in some constant.

Reply:
like u want me to use data binding for that?

Reply:
yes